tests/e2e/test_migrating/migration_signed_invokes.cairo

```diff
@@ -3,7 +3,8 @@
 @external
 func up() {
     %{
-        contract  = deploy_contract("./build/main.json")
+        declaration = declare("./build/main.json", config={"max_fee": "auto"})
+        contract = deploy_contract(declaration.class_hash, config={"max_fee": "auto"})
```

Comment:
Shouldn't `deploy_contract` (and the new `deploy` command) run `declare` under the hood?

Reply:
I don't think it should, since we don't want to declare the contract each time we deploy it (for example for performance reasons, or costs).